### PR TITLE
feat: implement mod resource (#250)

### LINKED ISSUE #250: implement `mod` resource (% operator)
If we have division and multiplication operators, we should also have a modulo operator. However, currently it does not exist.

```
🦄 40/6
float ask(call(get('/'),40,6))
6.666666666666667
🦄 40*6
int ask(call(get('*'),40,6))
240
🦄 .editor
// Entering editor mode (^D to finish, ^C to cancel)
40%6

Uncaught Error: Unknown identifier '%'!
    at Object.<anonymous> (/Users/milimetr/Desktop/x/askql/dist/askvm/lib/run.js:70:15)
    at Generator.next (<anonymous>)
    at /Users/milimetr/Desktop/x/askql/dist/askvm/lib/run.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/milimetr/Desktop/x/askql/dist/askvm/lib/run.js:4:12)
    at Object.run (/Users/milimetr/Desktop/x/askql/dist/askvm/lib/run.js:29:12)
🦄
```

This task is to create a `mod` resource and a `%` operator linked to this resource. 

-- 

Dear new contributor,

If you have any questions on how to start, please write a comment with a tag: @czerwinskilukasz1 or @mhagmajer, or join our Discord Community and let us know there. We will be very happy to help you start! :)
Hint for starters: As a starting point, take a look at `src/askvm/resources/math/index.ts` file, where a `*` operator and `times` resource are already linked or a couple of other ones and use them as an example.

Cheers,
Łukasz (@czerwinskilukasz1)
AskQL Core Developer

### PR DESCRIPTION
**Pull request by develper**
Closes #250

### CODE REVIEW

File: src/askscript/__tests__/09-operators/operators-10-mod-oper.test.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { resources } from '../../../askvm';
+import { e2e } from '../../../utils/tools';
+
+test('mod operator', async () => {
+  const environment = { resources };
+  await expect(e2e('1998%9', environment)).resolves.toBe(0);
```
Comment (czerwinskilukasz1) [inline src/askscript/__tests__/09-operators/operators-10-mod-oper.test.ts:6]: Your tests are good, could you just add spaces around `%` signs to improve readability?

Comment (develper) [reply src/askscript/__tests__/09-operators/operators-10-mod-oper.test.ts]: > package.json and playgroundBuild.ts shouldn't be changed in this PR as these are changes unrelated to modulo
The issue was that my forked repo was not sync with xFAANG/askql. 
> Your tests are good, could you just add spaces around % signs to improve readability?
done that.

**Comment by km4** [comment]
now it's fine ! grate job 👍

**Comment by czerwinskilukasz1** [comment]
Looks good. Bhargav, thank you for your next PR! 🙂   @develper

**Pull request merged**